doc/tutorial.txt

```diff
@@ -422,6 +422,39 @@ following sequence:
 	"unlock" : "mutexA"
 }
 
+* barrier : String. Used as at least a pair where the name must match.
```

Comment:
Yes, if you only have one user that user is guaranteed to get stuck there. I will add that.